### ISSUE TITLE
Tests should run on installed version

### ISSUE THREAD

**Issue by paulray** [nanograv/PINT#268]
It seems that when travis-ci runs `make test` it imports the local version of pint, rather than installing it an using the installed version.  This makes the test incomplete since errors in the install script won't be caught (most recently, the tt2tai_bipm2015.clk file was left out of setup.py so it didn't get copies  to the installed datafiles dir).  @cdeil or @jellis18 do you know how to address this?

**Comment by cdeil** [comment]
This should install PINT and then run the tests with the installed version, no?

To install
```
pip install .
```
or this should be equivalent
```
python setup.py install
```

Then cd into the tests folder to avoid the local version being picked up:
```
cd tests
```
and just run the tests like this
```
nosetests
```
or this
```
python -m pytest
```

Some packages (like Numpy, Scipy, Astropy, ...) put the tests in the package, so that one can also run them from the installed version. That would be a larger change, so if you like it, I would suggest to do that as a second step.

**Comment by paulray** [comment]
Right, so should we change `.travis.yml` to do that?  When I look at it now, I don't see where it runs install at all.  I don't actually see how it works at all. When does it build the package?

**Comment by cdeil** [comment]
> Right, so should we change .travis.yml to do that?

yes

> When does it build the package?

I think `make test` triggers `python setup.py nosetests` which for some reason (which I didn't track down) triggers a build:
https://github.com/nanograv/PINT/blob/master/Makefile#L54
https://travis-ci.org/nanograv/PINT/jobs/214367794#L340

**Comment by paulray** [comment]
Fixed in #270
